packages/flex-plugin-webpack/src/webpack/webpack.config.ts

```diff
@@ -318,28 +319,41 @@ export const _getJSPlugins = (environment: Environment): Plugin[] => {
       }),
     );
   }
-  const hasPnp = 'pnp' in process.versions;
+  // const hasPnp = 'pnp' in process.versions;
 
   if (getPaths().app.isTSProject()) {
     const typescriptPath = resolveModulePath('typescript');
-    const config: Partial<ForkTsCheckerWebpackPlugin.Options> = {
-      typescript: typescriptPath || undefined,
+    // This entire Type changed and having trouble finding replacements to some attributes (commented below)
```

Comment:
[This link](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/plugin-options.json) shows you the new format of the type and [this](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/plugin-options.ts) is the interface.